### ISSUE TITLE
Issues with the website

### ISSUE THREAD

**Issue by wilsonsilva** [HellRok/Taylor#22]
- https://github.com/hellrok/mruby-require can't be found (mentioned in https://www.taylormadetech.dev/documentation/)
- The playground is not loading

**Comment by Chadowo** [comment]
On your second point: It's strange, the playground loads and works fine for me. I think it would be useful for Hellrok if you could share your OS and browser + version (make sure to check the developer's console for any errors).

**Comment by wilsonsilva** [comment]
@Chadowo I don't know what happened. I was getting a redirection to a page on port 4xxxx and the page was not loading before.

But it is working now.

**Comment by Chadowo** [comment]
I see, well I can't really think of any possible causes, but I'm glad it works now :).

Anyhow, your first point still stands, I think it's intended to link to [mattn/mruby-require](https://github.com/mattn/mruby-require) (the mgem that Taylor currently uses) but I can't be so sure.

**Comment by HellRok** [comment]
Hi!

I've fixed up the link to mruby-require, I was still linking to the fork I had to make to solve some issues a while back.

I'm not sure what could have happened with the playground issue you had, I haven't done a deploy before today for a fair while and it's just a static website served by nginx in a docker container, so not really much to go wrong.

Cheers for alerting me to things though and I hope you enjoy Taylor!
